toolkit/components/uniffi-bindgen-gecko-js/src/ci_list.rs

```diff
@@ -0,0 +1,122 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
bike-shed - `ci_list` isn't a great name - the "Universe of ComponentInterfaces" is a good description, so I'd almost see `uci` or `ciu` as a reasonable name - but as with all my comments here, take them for what they cost you - obviously changing filenames later is trivial.